core/storage/user/gae_models_test.py

```diff
@@ -94,25 +96,33 @@ def setUp(self):
             subject_interests=self.GENERIC_SUBJECT_INTERESTS,
             first_contribution_msec=1,
             preferred_language_codes=self.GENERIC_LANGUAGE_CODES,
-            preferred_site_language_code=(self.GENERIC_LANGUAGE_CODES[0]),
-            preferred_audio_language_code=(self.GENERIC_LANGUAGE_CODES[0])
+            preferred_site_language_code=self.GENERIC_LANGUAGE_CODES[0],
+            preferred_audio_language_code=self.GENERIC_LANGUAGE_CODES[0],
+            profile_name=self.GENERIC_PROFILE_NAME,
+            associated_profile_user_ids=self.ASSOCIATED_PROFILE_IDS
         ).put()
 
-    def test_get_deletion_policy(self):
+    def test_get_deletion_policy_is_delete(self):
         self.assertEqual(
             user_models.UserSettingsModel.get_deletion_policy(),
             base_models.DELETION_POLICY.DELETE)
 
-    def test_apply_deletion_policy(self):
+    def test_apply_deletion_policy_registered_users_are_deleted(self):
```

Comment:
I think usage of full sentences in the naming of tests everywhere will make it very long (longer than the 80 characters limit). I have still updated the name and hope it looks better.

Reply:
The name I suggested is only 1 character longer than the existing one, but if that causes issues, it is fine to omit "that".

If you write test names in small sentence fragments, it is hard to understand them. Past test names are not necessarily a good guide because we have only started being more rigorous about test names recently.

Reply:
Yes old tests are not good reference over what you are saying. Sorry I did not mean to ignore your suggestion. 
I was just confused which naming convention to follow - the one in the guide or what seems easy to read and understand. I myself preferred latter one. But I think the guide mostly wants to tell people not to miss any of the "3 things" in the name, their order should only be such that it makes it more readable.
I am pausing this change for a while, I have my 1:1 with @BenHenning today, I will get it clarified, and I will write the decision here and make the changes. @seanlip I hope you don't mind me doing that. I just wanted to understand it properly at once.